jakarta-eclipse/org.eclipse.lsp4jakarta.core/src/org/jakarta/jdt/jax_rs/Jax_RSConstants.java

```diff
@@ -30,7 +30,8 @@
     public static final String DIAGNOSTIC_SOURCE = "jakarta-jax_rs";
 
     /* Severity */
-    public static final DiagnosticSeverity SEVERITY = DiagnosticSeverity.Error;
+    public static final DiagnosticSeverity SEVERITY_ERROR = DiagnosticSeverity.Error;
+    public static final DiagnosticSeverity SEVERITY_WARNING = DiagnosticSeverity.Warning;
```

Comment:
Feels redundant to set these as constants in every constants class. Can we remove these from the constants class and instead just call `DiagnosticSeverity.Warning/Error` directly in the `completeDiagnostic` method below.

There are a number of other constants classes that do the same thing that we should fix.